test/module-test.cc

```diff
@@ -552,21 +505,27 @@ TEST(module_test, custom_context) {
   EXPECT_TRUE(!custom_arg);
 }
 
-struct disabled_formatter {};
-
-TEST(module_test, has_formatter) {
-  EXPECT_FALSE(
-      (fmt::has_formatter<disabled_formatter, fmt::format_context>::value));
-}
-
-TEST(module_test, is_formattable) {
-  EXPECT_FALSE(fmt::is_formattable<disabled_formatter>::value);
-}
-
 TEST(module_test, compile_format_string) {
   using namespace fmt::literals;
+#ifdef __clang__
+  fmt::println("\033[0;33m[=disabled=] {}\033[0;0m", "Clang 16.0 fails to import user-defined literals");
+#else
   EXPECT_EQ("42", fmt::format("{0:x}"_cf, 0x42));
   EXPECT_EQ(L"42", fmt::format(L"{:}"_cf, 42));
   EXPECT_EQ("4.2", fmt::format("{arg:3.1f}"_cf, "arg"_a = 4.2));
   EXPECT_EQ(L" 42", fmt::format(L"{arg:>3}"_cf, L"arg"_a = L"42"));
+#endif
 }
+
+TEST(module_test, std_types) {
+  EXPECT_EQ(R"("42")", fmt::format("{}", std::filesystem::path("42")));
+  EXPECT_EQ(R"(optional("42"))",
+            fmt::format("{}", std::optional<std::string>("42")));
+  EXPECT_EQ("none", fmt::format("{}", std::optional<std::string>()));
+  EXPECT_EQ("variant(42)",
+            fmt::format("{}", std::variant<int, std::string>(42)));
+  EXPECT_EQ("monostate", fmt::format("{}", std::monostate()));
+  EXPECT_EQ("system:0", fmt::format("{}", std::error_code()));
+  EXPECT_EQ("0", fmt::format("{}", std::thread::id()));
+  EXPECT_EQ("42", fmt::format("{}", std::runtime_error("42")));
+}
```

Comment:
I think we should stop adding new test cases to module-test.

Reply:
I can drop this for your repo and keep it in mine. I want tests **now**.